src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -29,8 +29,12 @@
 public abstract class CloudSdkMojo extends AbstractMojo {
 
   /** Optional parameter to configure the location of the Google Cloud SDK. */
-  @Parameter(property = "cloudSdkPath", required = false)
-  private File cloudSdkPath;
+  @Parameter(property = "cloudSdkHome", required = false)
```

Comment:
I wouldn't remove cloudSdkPath completely, maybe use the strategy used for appYamls in RunMojo that kind of shows a deprecation notice? It might not be best if we just swapped this name on users without warning.

Reply:
Gonna change it back for now.